server/package.json

```diff
@@ -49,9 +49,10 @@
     "react-dom": "^16.0.0",
     "react-json-view": "^1.13.2",
     "react-octicons": "^0.2.0",
-    "sass-loader": "^6.0.6",
-    "style-loader": "^0.19.0",
-    "webpack": "^3.8.1"
+    "sass-loader": "^7.1.0",
+    "style-loader": "^0.23.0",
+    "webpack": "^4.19.1",
+    "webpack-cli": "^3.1.0"
```

Comment:
I also had to install the [`webpack-cli`](https://github.com/webpack/webpack-cli) package in order for `npm run build` to work after upgrading.